src/index.ts

```diff
@@ -1,6 +1,8 @@
-import { createContext, runInContext } from 'js-slang'
+import { createContext, runInContext, Result as SourceResult } from 'js-slang'
 import { SourceError } from 'js-slang/dist/types'
 
+const TIMEOUT_DURATION = process.env.NODE_ENV !== 'test' ? 20000 : 1000
```

Comment:
Prefer if this can be an env variable instead.

Reply:
Fixed with 6d02b6a, but will need a complementary change in the terraform files of cadet to set this environment variable in the lambda node env. Will make this PR later.